api_tests/lib/ledger_runner.ts

```diff
@@ -0,0 +1,103 @@
+import * as bitcoin from "./bitcoin";
+import * as toml from "toml";
+import * as fs from "fs";
+import { sleep } from "./util";
+import { execSync, spawn } from "child_process";
+
+const project_root: string = execSync("git rev-parse --show-toplevel", {
+    encoding: "utf8",
+}).trim();
+const docker_cwd = project_root + "/api_tests/regtest";
+
+export class LedgerRunner {
+    private running_ledgers: { [key: string]: boolean };
+    private readonly block_timers: { [key: string]: NodeJS.Timeout };
+    private readonly docker_compose_options: { cwd: string; encoding: string };
+    private readonly docker_cwd: string;
+    private readonly log_dir: string;
+    private readonly ledgers_config: any;
+
+    constructor(
+        project_root: string,
```

Comment:
I'd just pass in `docker_cwd` instead of this.
It may be nice to rename it to something like `docker_compose_dir` since that's what it really means.
Even better would be to make it `docker_compose_file` and have it be the path to the `docker-compose.yml`. Then we could stop setting the `cwd` everywhere and just use the `-f` flag to `docker-compose` to set the config file to use.

Reply:
I like that idea of having passing just docker-compose file. Will do so.